developer_manual/digging_deeper/groupware/contacts_menu.rst

```diff
@@ -0,0 +1,89 @@
+=============
+Contacts Menu
+=============
+
+Nextcloud shows a *Contacts menu* in the right corner of the header. This menu lists a user's contacts. These contact entries can be extended by apps.
```

Comment:
An image (as an example might be nice here).

Reply:
I'm not a fan of too many screenshots in the docs because they are getting outdated so fast and redoing all screenshots for a release is a lot of work

Reply:
It was just a suggestion. Feel free to resolve this comment.